SU2_CFD/src/numerics_direct_mean_inc.cpp

```diff
@@ -500,7 +593,35 @@ void CCentLaxInc_Flow::ComputeResidual(su2double *val_residual, su2double **val_
       }
     }
   }
-  
+
+  /*--- Jacobian contributions due to grid motion ---*/
```

Comment:
Can you please double-check some of these comments? They are a little misleading now, since it says only Jacobian contributions but is also makes residual changes. It might be one of my old comments that just didn't get updated during changes..

Reply:
The comments have been updated now for FDS, JST and LAX to make it clear when the residual and Jacobian contributions due to grid motion are handled